src/index.js

```diff
@@ -32,6 +32,11 @@ fs.readdirSync(commandsDir).forEach((command) => {
 	lisky.use(commandModule.default);
 });
 
+const copyright = `Lisky  Copyright (C) 2017  Lisk Foundation
```

Comment:
Is the odd spacing deliberate?

Reply:
This (also the capital C) was proposed from the actual license. To distinguish the three information. Project, Copyright, Fullname